packages/rspack/scripts/install.mjs

```diff
@@ -0,0 +1,17 @@
+import { execSync } from "child_process";
+
+function getVersion(packageName) {
+	const command = `npm show ${packageName} version`;
+	return execSync(command).toString().trim();
+}
+
+const coreVersion = getVersion("@rspack/core");
+const bindingVersion = getVersion("@rspack/binding");
+
+if (coreVersion !== bindingVersion) {
+	throw new Error(
+		`Expected ${JSON.stringify(coreVersion)} but got ${JSON.stringify(
```

Comment:
The msg isn't clear. Can you update to `The rspack core version xx is not match to binding version xx`. cc @hardfist

Reply:
> The msg isn't clear. Can you update to `The rspack core version xx is not match to binding version xx`. cc @hardfist

wait a minute